devel/python/CMakeLists.txt

```diff
@@ -7,22 +7,23 @@ if (NOT DEFINED PYTHON_EXECUTABLE)
    return()
 endif()
 
+python_module(cwrap REQUIRED )
 
-python_module(numpy 1.7.1)
+python_module(numpy REQUIRED 1.7.1)
```

Comment:
Not 100% clear to me what is the best way to do this, but the requirements are:

1. It should be possible and valid to build a C only target.
2. The python target `ert.ecl` *requires* `numpy` and  `cwrap`.

To get graceful degradation in the case where the user has requested Python bindings, but does not have `numpy`or `cwrap` we can not have `REQUIRED` on `numpy` and `cwrap`.